RealmSwift/List.swift

```diff
@@ -509,14 +484,165 @@ extension List: RealmCollection, RangeReplaceableCollection {
     /// :nodoc:
     public func _observe(_ block: @escaping (RealmCollectionChange<AnyRealmCollection<Element>>) -> Void) ->
         NotificationToken {
-        let anyCollection = AnyRealmCollection(self)
-        return _rlmArray.addNotificationBlock { _, change, error in
-            block(RealmCollectionChange.fromObjc(value: anyCollection, change: change, error: error))
+            let anyCollection = AnyRealmCollection(self)
```

Comment:
The indentation here looks a bit odd. The function body appears to be indented one level too far.